gulpfile.js

```diff
@@ -1,6 +1,8 @@
 /* global __dirname */
 
-(function (pkg, gulp, jshint, jscs, stylish, mocha, connect, less, requirejs, uglify, replace, header, awspublish, aws, yargs, rimraf, mkdirp, recursive) {
+(function (pkg, gulp,
```

Comment:
This line was getting too long so I've broken it up into chunks.  I put `pkg` and `gulp` on their own line, then all the `gulp` plugins, then other stuff.